ompi/tools/wrappers/ompi_wrapper_script.in

```diff
@@ -66,7 +66,7 @@ my $linker_flags = $libdir_flag . $libdir . " " . $extra_ldflags;
 # pulled in implicitly) because we intend MPI applications to only use
 # the MPI API.
 my $libs = "-l".$ompi_libmpi_name." " . $extra_libs;
-my $libs_static = "-l".$ompi_libmpi_name." -lopen-rte -lopen-pal " . $extra_libs;
+my $libs_static = "-l".$ompi_libmpi_name." -lopen-pal " . $extra_libs;
```

Comment:
This is also wrong in the same way as the .pc files.

Reply:
This is done in a lotta places, worthy of its own commit at least (or a seperate PR). Can you point me to what it should be? Checking the Makefile.am it isn't obvious.

Reply:
@awlauria I think Brian is referring to two things:

1. The `@OMPI_LIBMPI_NAME@` stuff in `ompi/Makefile.am`, which is tied to `--with-libmpi-name=FOO`.
1. The `@OPAL_LIB_PREFIX@` stuff in `opal/Makefile.am`.

However, I can't find a `configure` CLI option to actually set the `OPAL_LIB_PREFIX` value.  https://github.com/open-mpi/ompi/blob/19b9b7f0a597b53e42d58d00fc1487a32ef06f96/configure.ac#L331 This ^^ is a hard-coded call to set `OPAL_LIB_PREFIX` to an empty string in `configure.ac`.  I can't find anywhere else that sets `OPAL_LIB_PREFIX` to another value.

Sidenote: IIRC, this kind of functionality was put there for Spectrum MPI to be able to rename its libraries in a branded way (and/or have multiple libraries installed to the same tree, but the libraries would have different filenames, and therefore the wrapper compiler and/or mpirun could select which one to use based on filename).  This *might* have been related to having multiple MPI Fortran binding libraries available simultaneously...?  I don't remember.  🤷‍♂️ 

Do you know if this functionality is still used?

If we now *always* name the OPAL library a fixed name, that's some more dead code that could be eliminated.

Ditto for if Spectrum no longer needs a variable filename for the MPI library.

I know you tried to do a nice small thing; sorry the scope keeps creeping up on you...  That's the Open MPI Way(tm).  😄

Reply:
Thanks. Why have an `OPAL_LIB_PREFIX`? Why not just `OPAL_LIB_NAME` to match OMPI?

Reply:
Perhaps that day is today. PR updated.

Reply:
Spectrum MPI still uses the `--with-libmpi-name` option to change the name of the main MPI library (`libmpi_ftw`). We currently test this configure option in IBM's MTT to make sure it doesn't break.
 * `-with-libmpi-name="mpi_ftw"` as seen [here](https://mtt.open-mpi.org/index.php?limit=&wrap=&trial=&enable_drilldowns=&yaxis_scale=&xaxis_scale=&hide_subtitle=&split_graphs=&remote_go=&do_cookies=&phase=mpi_install&text_start_timestamp=2021-05-19+21%3A24%3A27+-+2021-05-20+21%3A24%3A27&text_platform_hardware=%5Eppc64le%24&show_platform_hardware=show&text_os_name=%5ELinux%24&show_os_name=show&text_mpi_name=%5Eompi-nightly-master%24&show_mpi_name=show&text_mpi_version=%5Emaster-202105200241-19b9b7f%24&show_mpi_version=show&text_configure_arguments=&text_compiler_name=%5Egnu%24&show_compiler_name=show&text_bitness=%5E64%24&show_bitness=show&text_endian=%5Eunknown%24&show_endian=show&text_http_username=%5Eibm%24&show_http_username=show&text_local_username=all&show_local_username=hide&text_platform_name=%5Eibm-community%24&show_platform_name=show&click=Detail&phase=mpi_install&test_result=_mpi_p&text_compute_cluster_id=&text_os_version=&show_os_version=&text_platform_type=&show_platform_type=&text_submit_id=&text_hostname=&show_hostname=&text_mpi_get_id=&text_mpi_install_compiler_id=&text_mpi_install_configure_id=&text_mpi_install_id=&text_compiler_version=&show_compiler_version=&text_vpath_mode=&show_vpath_mode=&text_exit_value=&show_exit_value=&text_exit_signal=&show_exit_signal=&text_duration=&show_duration=&text_client_serial=&show_client_serial=&text_result_message=&text_result_stdout=&text_result_stderr=&text_environment=&text_description=&text_trial=%5Ef%24&lastgo=summary)